content/specific-skill-success-criteria/for-loops/_index.md

```diff
@@ -26,7 +26,7 @@ for (some_initialiser,some_check,some_update){
 ```
 They should be able to reason about each of the different parts of the configuration of the loops.
 
-\## Python learners understand `range` and `in`.
+### Python learners understand `range` and `in`.
```

Comment:
Why is this heading nested under:  `## JS and Java learners understand that they can count up and down.` 
The hierarchy makes no sense.

These are distinct concepts

Reply:
Either leave everything as heading level 2 or:

Make a new heading level 2 `## Concepts` and nest the different concepts under that.

Please try to put effort into understanding heading hierarchies.

It's like this:

```
## x
## y
### z
### a
```

This implies a tree structure:

```
x 

y
   z
   a
```

z and a are subheadings in the section titled y
z and a are inside y 

You are saying "All learners understand that the whole body of the loop executes on each iteration." is inside "JS and Java learners understand that they can count up and down" which is cleartly wrong. These things are siblings, not parent child.

Please don't submit this kind of thing again. If you are still unclear on heading hierarchies I'm sure Whitney will be able to explain them to you. Never guess when submitting work. I simply will not merge things that are so clearly incorrect.